scripts/setup.sh

```diff
@@ -1,27 +1,26 @@
 #!/usr/bin/env bash
 npm install
-git clone https://github.com/graphhopper/graphhopper.git
-git clone https://github.com/graphhopper/map-matching.git
-# cd graphhopper
-# git checkout 0.10
-# cd ..
+git clone -b 0.10 https://github.com/graphhopper/graphhopper.git
+git clone -b 0.9 https://github.com/graphhopper/map-matching.git
+
 mkdir graphhopper-walking
 cd graphhopper-walking
-git clone https://github.com/graphhopper/graphhopper.git
-# cd graphhopper
-# git checkout 0.10
+git clone -b 0.10 https://github.com/graphhopper/graphhopper.git
 cd ..
 
 cd map-matching
-git checkout 0.9
 ./map-matching.sh action=import datasource=../osrm/map.osm vehicle=car,foot
 cd ..
+
 wget https://s3.amazonaws.com/tcat-gtfs/tcat-ny-us.zip
+
 mkdir tcat-ny-us
 tar xvf tcat-ny-us.zip -C tcat-ny-us
+
 cd graphhopper
 ./graphhopper.sh buildweb
```

Comment:
would this work if you did `./graphhopper/graphhopper.sh buildweb` without the `cd`?